extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -84,6 +86,16 @@ export class ConsentUI {
       config['uiConfig'] &&
       config['uiConfig']['overlay'] === true;
 
+    /** @private {string} */
+    this.consentTitle_ =
+      (config['captions'] && config['captions']['consentTitle']) ||
+      'User Consent Prompt';
+
+    /** @private {string} */
+    this.buttonTitle_ =
+      (config['captions'] && config['captions']['buttonTitle']) ||
```

Comment:
nit: Would be great to define the default message in a const string.

Reply:
`buttonTitle` is a bit confusing to me, it's an invisible button for screen reader only. `captions` kind of convey that message, but I am not sure if developer understand what to put there. 
I don't have a good name in mind though. any idea?

Reply:
Changed to `consentPromptCaption_` and `buttonActionCaption_`, I think this is more clear.